apps/src/code-studio/components/ImageURLInput.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import moment from 'moment';
+import InputPrompt from '@cdo/apps/templates/InputPrompt';
+import i18n from '@cdo/locale';
+import {ABSOLUTE_REGEXP} from '@cdo/apps/assetManagement/assetPrefix';
+
+const styles = {
+  supportingText: {
+    margin: '1em 0',
+    fontSize: '16px',
+    lineHeight: '20px'
+  },
+  example: {
+    margin: '1em 0',
+    fontSize: '16px',
+    lineHeight: '20px'
+  },
+  error: {
+    color: 'red'
+  }
+};
+
+export default class ImageURLInput extends React.Component {
+  static propTypes = {
+    assetChosen: PropTypes.func,
+    allowedExtensions: PropTypes.string,
+    currentValue: PropTypes.string
+  };
+  constructor(props) {
+    super(props);
+    this.state = {showError: false};
+  }
```

Comment:
since we're only using the constructor to set initial state, we can remove the constructor and set state directly in the component scope like this example:

https://github.com/code-dot-org/code-dot-org/blob/d19bee2aef790bb963498080686d7fcabcbdf9fe/apps/src/code-studio/components/libraries/LibraryCreationDialog.jsx#L83-L90